frontend/src/components/Cell.tsx

```diff
@@ -28,18 +28,23 @@ export const Cell = ({
         on {`${date}`}
       </label>
       <input
-        type="number"
+        type="text"
         id={`${topic.issue.id}${topic.activity.id}${formatDate(
           date,
           "yyyy-MM-dd"
         )}`}
-        min={0}
         onChange={(event: any) => {
+          //makes sure that users can only input positive numbers up to 999.99999999...
+          //with an unlimited number of decimals behind the delimiter
+          event.target.value = event.target.value
+            .replace(/[^0-9.]/g, "")
+            .replace(/(^\d{3})\d/g, "$1")
+            .replace(/(\..*?)\..*/g, "$1");
```

Comment:
I'm assuming this trims further dots off from the string.

I'm uncertain how that expression works, but you could also just match a dot followed by any number on non-dots, and then delete the rest of the string after that:
```
.replace(/(\.[^.]*).*$/, "$1")
```

Reply:
I tested this and it doesn't allow you to write decimal numbers, because it removes _everything_ after the first dot.

Reply:
Ow, yes.